news/_posts/2024-05-05-ilastik-1.4.1.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "New stable ilastik release 1.4.1"
+---
+
+We are happy to announce the release of the new stable [ilastik version `1.4.1`]({{site.baseurl}}/download.html#stable).
+
+Among many other improvements, this release contains [support for OME-zarr multiscale images]({{site.baseurl}}/documentation/basics/dataselection#multiscale).
+You can now load and process selected scales of OME-zarr images (and other multiscale formats like Neuroglancer precomputed chunks, and N5).
```

Comment:
```suggestion
You can now load and process selected scales of OME-Zarr images (and Neuroglancer precomputed chunks). Please get in touch if support for other multiscale formats would help your work.
```

I guess if you had some custom multiscale N5 store, you can load individual scales as you can load any arbitrary N5 dataset. It wouldn't work through the "Add multiscale dataset" flow though.

Reply:
Ha thanks for catching that!